doc/Programming-guide/Key-platform-features/Endpoint-groups/index.md

```diff
@@ -6,166 +6,308 @@ nav: /:path/Programming-guide/Key-platform-features/Endpoint-groups
 sort_idx: 30
 ---
 
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
 
-Kaa allows for aggregating endpoints related to the same application into endpoint groups. The endpoint group represents an independent managed entity which is defined by the profile filters assigned to the group. Those endpoints whose profiles match the profile filters of the specific endpoint group become automatically registered as members of this group. There is no restriction for endpoints on having membership in a number of groups at a time.
-Endpoint group profile filters are predicate expressions which define characteristics of group members (endpoints). These filters are executed against the endpoint profile to figure out whether or not the endpoint belongs to the group.
+* TOC
+{:toc}
 
-### NOTE ###
+Kaa provides a mechanism for endpoints aggregation within the application that is based on groups. Grouping endpoints enables you to activate specific configuration parameters, 
+control access to notification topics, etc.
+This guide will familiarize you with the basic concepts of designing endpoint groups. It is assumed that you have either set up a Kaa Sandbox, or a fully-blown Kaa cluster 
+already and that you have created a tenant and an application in Kaa. It is also strongly recommended to review 
```

Comment:
that you have created a tenant and an application in Kaa.

Can here be added a link to the part that describes how to achieve this requirement?